### ISSUE TITLE
Audit file warrning in logs 

### ISSUE THREAD

**Issue by mjudeikis** [openshift/openshift-azure#1163]
```
W0218 11:59:19.677033       1 start_master.go:289] Warning: auditConfig.auditFilePath: Required value: audit needs to be logged to a separate file, master start will continue.
```

/cc @thekad @jim-minter 

I think we should move audit logs to separate file and read it separately. easier to handle file retention too. 
WDYT guys?

**Comment by thekad** [comment]
We specifically went with shipping everything to journald so we didn't have to re-invent the log rotation/retention wheel, however I am feeling like this could be a better approach, and upstream seems to feel that way as well with that warning.

**Comment by thekad** [comment]
Should be fixed by #1455
